SPECS/xz/xz.spec

```diff
@@ -105,6 +107,9 @@ make  %{?_smp_mflags}  check
 %defattr(-,root,root)
 
 %changelog
+* Tue Apr 1 2025 Jon Slobodzian <joslobo@microsoft.com> - 5.4.4-2
```

Comment:
Can we change to Apr 01?

Reply:
Probably not - this is the way it's already been published. This PR is just a sync of changes.

Reply:
No problem, this is just a minor consistency issue